src/lambda_functions/get_prices_lambda/scraper.py

```
@@ -561,202 +593,148 @@
+        try:
+            urls = self.config['urls']['official']
+            with ThreadPoolExecutor(max_workers=MAX_WORKERS) as executor:
+                futures = [executor.submit(self.scrape_url, url) for url in urls]
+                results = []
+                for future in as_completed(futures):
+                    try:
+                        results.extend(future.result())
+                    except Exception as e:
+                        self.error_handler.handle_error(ScraperError(str(e)))
+                return results
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)